### PR TITLE
Clean up libjdwp per reviewers' comments

### PR DESCRIPTION
**Pull request by pdbain-ibm**
Remove unused function and redundant variable settings.
Clean up bad indenting.

Signed-off-by: Peter Bain <peter_bain@ca.ibm.com>

### CODE REVIEW

**Comment by pdbain-ibm** [comment]
@andrew-m-leonard and @pshipton please review.

File: jdk/src/jdk.jdwp.agent/share/native/libjdwp/debugInit.c [hunk 1]
```diff
@@ -819,7 +810,7 @@ debugInit_reset(JNIEnv *env)
 
     currentSessionID++;
     initComplete = JNI_FALSE;
-    VMInitComplete = JNI_TRUE; /* The VM Is already initialized */
+    /* The VM Is already initialized so no need to set VMInitComplete = JNI_TRUE. */
```
Comment (pshipton) [inline jdk/src/jdk.jdwp.agent/share/native/libjdwp/debugInit.c]: I don't think we want this comment in the code as it becomes another patch to maintain as the original code changes.

Comment (pdbain-ibm) [reply jdk/src/jdk.jdwp.agent/share/native/libjdwp/debugInit.c]: Okay.  Comment removed.

**Review by pshipton — APPROVED** [review]

**Review by andrew-m-leonard — APPROVED** [review]
Looks good

**Comment by pdbain-ibm** [comment]
Same change for Java 8 https://github.com/ibmruntimes/openj9-openjdk-jdk8/pull/70

**Pull request merged**